src/ApiManagement/ApiManagement/help/New-AzApiManagementBackend.md

```diff
@@ -154,8 +154,7 @@ Accept wildcard characters: False
 
 ### -ResourceId
 Management Uri of the Resource in External System.
-This parameter is optional.
-This url can be the Arm Resource Id of Logic Apps, Function Apps or Api Apps.
```

Comment:
Please update the description in https://github.com/Azure/azure-powershell/blob/main/src/ApiManagement/ApiManagement.ServiceManagement/Commands/NewAzureApiManagementBackend.cs#L64

Reply:
Done